.github/workflows/cpp-packaging.yml

```diff
@@ -544,6 +549,9 @@ jobs:
           cd sdk-src
           python scripts/gha/install_prereqs_desktop.py --ssl boringssl
           cd ..
+          if [[ $(uname) == "Darwin"* ]]; then
```

Comment:
Fixes an issue where Mac packaging was not done in parallel due to the "parallel" tool not being installed. Speeds up packaging on Mac.

Reply:
:+1: